book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] please ensure Getters and Setters for @rentals should be present
eg   `attr_accessor :title, :author, :rentals`